packages/components/radio-group/src/RadioGroup.doc.mdx

```diff
@@ -39,6 +39,10 @@ import { RadioGroup } from '@spark-ui/radio-group'
 
 <Canvas of={RadioStories.Default} />
 
+## With a multiline label
+
```

Comment:
Don't you think we should be more explicit about the limitations concerning label line-height? As said within original issue it could seem obvious and out of our scope to care about these bad implementations...
But it might also be cool to add a small reminder about it (not using headings sizes for labels). Wdyt?

Reply:
Yep, adding it is worth it 😉